### PR TITLE
feat(yabloc): add yabloc trigger service to suspend and restart the estimation

### PR DESCRIPTION
**Pull request by KYabuuchi**
## Description

This PR adds a service to stop and resume yabloc.
This was created to split the PR https://github.com/autowarefoundation/autoware.universe/pull/5846

### Minor change
* I have also changed the inappropriate default value for `resized_image`. This change does not affect the behavior since it is overwritten by the upper launch file.

## Tests performed

The following data sets were used to confirm the behavior. 
(We can conrifm the behavior of this [even with the sample rosbag in logging_simulator.](https://autowarefoundation.github.io/autoware-documentation/main/tutorials/ad-hoc-simulation/rosbag-replay-simulation/))

* [map](https://github.com/tier4/AWSIM/releases/download/v1.1.0/nishishinjuku_autoware_map.zip) : This link was quoted from [AWSIM tutorial](https://tier4.github.io/AWSIM/GettingStarted/QuickStartDemo/#start-the-demo).
* [rosbag](https://drive.google.com/file/d/1zKTGRH4lD-wptpOdNCgpiPfGRDP0XrUm/view?usp=sharing)

```bash
ros2 launch autoware_launch logging_simulator.launch.xml \
    map_path:=/path/to/nishishinjuku  \
    vehicle_model:=sample_vehicle \
    sensor_model:=awsim_sensor_kit
    pose_source:=yabloc

ros2 bag play yabloc_autoware_test_made_in_awsim
ros2 service call /localization/pose_estimator/yabloc/pf/yabloc_trigger_srv std_srvs/srv/SetBool data: false
# We can see "yabloc particle filter is deactivated" in console.
# In the inactive state,  /localization/pose_estimator/pose_with_covariance is NOT published.

ros2 service call /localization/pose_estimator/yabloc/pf/yabloc_trigger_srv std_srvs/srv/SetBool data: true
# We can see "yabloc particle filter is activated" in console.
# In the active state,  /localization/pose_estimator/pose_with_covariance is published.
```

## Effects on system behavior

This PR does not change the localization accuracy.

Users can stop & restart YabLoc by callaing the trigger_service like following:
```
ros2 service call /localization/pose_estimator/yabloc/pf/yabloc_trigger_srv std_srvs/srv/SetBool data: true
ros2 service call /localization/pose_estimator/yabloc/pf/yabloc_trigger_srv std_srvs/srv/SetBool data: false
```

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by SakodaShintaro — APPROVED** [review]
Looks Good To Me

I was able to evaluate this PR using a rosbag data with GT from AWSIM, which I often use.
* Before : 0.681 m
* After : 0.675 m

**Comment by KYabuuchi** [comment]
@Motsu-san Please approve this as a code owner :pray:

File: localization/yabloc/yabloc_image_processing/launch/image_processing.launch.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <launch>
   <arg name="src_image" default="/sensing/camera/traffic_light/image_raw"/>
   <arg name="src_info" default="/sensing/camera/traffic_light/camera_info"/>
-  <arg name="resized_image" default="/sensing/camera/undistorted/image_raw/relay"/>
+  <arg name="resized_image" default="/sensing/camera/undistorted/image_raw"/>
```
Comment (Motsu-san) [inline localization/yabloc/yabloc_image_processing/launch/image_processing.launch.xml:4]: Is this change related to this pull request aim?

**Review by Motsu-san — COMMENTED** [review]
[LGTMeow](https://lgtmeow.com)
And I will appreciate if you answer my question.

**Review by Motsu-san — APPROVED** [review]

File: localization/yabloc/yabloc_image_processing/launch/image_processing.launch.xml (hunk 1 above)
Comment (KYabuuchi) [reply localization/yabloc/yabloc_image_processing/launch/image_processing.launch.xml:4]: Sorry. This line is a minor change not directly related to the main objective. But it does not change the system behavior.
 I added this modification to the description of PR.

image
